CIP-0068/README.md

```diff
@@ -171,15 +171,16 @@ This is a low-level representation of the metadata, following closely the struct
 
 ```
 ; Explanation here: https://developers.cardano.org/docs/native-tokens/token-registry/cardano-token-registry/
+; NOTE: 'logo' is an exception and is not a PNG bytestring, but a URI, so that links and data uris with all image types are possible (see below)
```

Comment:
neat-picking: why specifically _png_ ? I assume the intent is to forbid inline image data payload, regardless of their mime type.